Fix: APP-2347 - Next button disabled on membership type switch

Fabricevladimir opened the pull request:
## Description

- fixes issue of next button being disabled when toggling from multisig to token voting

Task: [APP-2347](https://aragonassociation.atlassian.net/browse/APP-2347)

## Type of change

<!--- Please delete options that are not relevant. -->

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.


[APP-2347]: https://aragonassociation.atlassian.net/browse/APP-2347?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

RuggeroCino commented:
@Fabricevladimir The build is currently failing, something's wrong on the `createDao.tsx` file

Fabricevladimir commented:
> @Fabricevladimir The build is currently failing, something's wrong on the `createDao.tsx` file

Thanks for catching that; I've fixed the issue and rebased.

RuggeroCino reviewed (approved):
Thanks @Fabricevladimir, LGTM!
